packages/vats/test/test-lib-chainStorage.js

```diff
@@ -3,8 +3,9 @@ import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
 
 import { makeChainStorageRoot } from '../src/lib-chainStorage.js';
 
-test('makeChainStorageRoot', async t => {
-  // Instantiate chain storage over a simple in-memory implementation.
+// makeFakeChainStorageKit creates a chainStorage root node over an in-memory map
+// and exposes both the map and the sequence of received messages.
+const makeFakeChainStorageKit = (rootPath, options) => {
```

Comment:
We have more than one kind of fake storage. How about `makeTransientChainStorageKit`?  Not sure we even need "chain" https://github.com/Agoric/agoric-sdk/pull/5945

Reply:
Also, WDYT of this in `lib-chainStorage` for use by other packages?